source/tools/cdk-integ-tools/package.json

```diff
@@ -31,9 +31,9 @@
     "typescript": "~3.9.7"
   },
   "dependencies": {
+    "@aws-cdk/assert": "0.0.0",
```

Comment:
Revert back

Reply:
reverted back to the version from master.